src/js/comments.js

```diff
@@ -26,8 +26,18 @@ class Comments {
 		/*global Coral*/
 		fetch('https://comments-auth.ft.com/v1/jwt/', {
 			credentials: 'include'
-		}).then(res => res.json())
-			.then(json => json.token)
+		}).then(response => {
+			if(response.ok) {
+				return response.json()
+			}
+			throw new Error('Bad response from the authentication service')
```

Comment:
Feel free to make these error messages more clear! I always have a tough time with them.